backend/daemon/daemon.go

```diff
@@ -513,7 +526,7 @@ func newNavigationHandler(router *mux.Router) http.Handler {
 	var routes []string
 
 	err := router.Walk(func(route *mux.Route, router *mux.Router, ancestors []*mux.Route) error {
-		u, err := route.URL()
+		u, err := route.URL("cid", "bafkreih7mye6mc5nux7oclgmopmo264plkc3paafivulbeyrzashzc2npy")
```

Comment:
@juligasa What does this supposed to do? It seems like for each route defined in the router this additional CID route gets added, and the actual URL is not being added for the navigation handler to display?

Reply:
This was needed in order for the nav bar function not to panic. Since the ipfs route has a variable param (`ipfs/{cid}`) it should be defined to show properly in the navbar. It only applies to the navbar
![image](https://user-images.githubusercontent.com/11684004/232454570-1150e4fa-3563-4409-bdfe-9111ec152e47.png)